runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatIndicator.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2018 Sebastiaan <https://github.com/SebastiaanVanspauwen>
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+ * ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+package net.runelite.client.plugins.clanchat;
+
+import java.awt.Color;
+import java.awt.image.BufferedImage;
+import net.runelite.api.Client;
+import net.runelite.client.ui.overlay.infobox.InfoBox;
+import net.runelite.client.ui.overlay.infobox.InfoBoxPriority;
+
+class ClanChatIndicator extends InfoBox
+{
+	private final ClanChatPlugin plugin;
+	private final ClanChatConfig config;
+	private final Client client;
+
+	ClanChatIndicator(BufferedImage image, ClanChatPlugin plugin, ClanChatConfig config, Client client)
+	{
+		super(image, plugin);
+		this.plugin = plugin;
+		this.config = config;
+		this.client = client;
+		setPriority(InfoBoxPriority.MED);
+		setTooltip(Integer.toString(plugin.getClanAmount()) + " clan member(s) near you");
```

Comment:
you should probably override getTooltip instead

Reply:
fixed